src/backend/tests/materials.controllers.test.ts

```diff
@@ -0,0 +1,11 @@
+const request = require('supertest');
+const express = require('express');
+const app = express();
+
+app.use(express.urlencoded({ extended: false }));
+
+test('materisal not found', async () => {
```

Comment:
you should add these tests to the projects test file, see how we test the Delete Team endpoint in `src\backend\tests\teams.test.ts` to see how we test a delete method without the corresponding create like you were mentioning